core/creator/src/main/java/io/quarkus/creator/phase/runnerjar/RunnerJarPhase.java

```diff
@@ -229,7 +248,9 @@ private void buildRunner(FileSystem runnerZipFs, CurateOutcome curateOutcome, Au
         final Map<String, Set<AppDependency>> duplicateCatcher = new HashMap<>();
         final StringBuilder classPath = new StringBuilder();
         final Map<String, List<byte[]>> services = new HashMap<>();
-
+        final Set<String> finalIgnoredEntries = Stream
+                .concat(IGNORED_ENTRIES.stream(), this.userConfiguredIgnoredEntries.stream())
+                .collect(toSet());
```

Comment:
```java
Set<String> finalIgnoredEntries = new HashSet<>(IGNORED_ENTRIES);
finalIgnoredEntries.addAll(this.userConfiguredIgnoredEntries);
```

would do the same trick, no? :-)

Reply:
Yes it will. 
I'm preparing another PR that will replace this one. I'll include this change in the new PR.